### PR TITLE
hosting: implement a console host

### PR DESCRIPTION
**Pull request by michelmegens**
Implement a console host in order to improve the experience when running the DSMR parser service as a console application. The console host uses a manual cancellation event to allow the user to easily stop the application using [CTRL]+C.

### CODE REVIEW

File: SensateIoT.SmartEnergy.Dsmr.Parser.Service/Application/ConsoleHost.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
+﻿/*
+ * TriggerService application runner.
+ *
+ * @author Michel Megens
+ * @email  michel@michelmegens.net
+ */
+
+using System;
+using System.Threading;
```
Comment (sensateiot-cicd) [inline SensateIoT.SmartEnergy.Dsmr.Parser.Service/Application/ConsoleHost.cs:9]: Add new lines.

**Review by sensateiot-cicd — CHANGES_REQUESTED** [review]
Nitpicking: add some newlines.

File: SensateIoT.SmartEnergy.Dsmr.Parser.Service/Application/ConsoleHost.cs (hunk 1 above)
Comment (michelmegens) [reply SensateIoT.SmartEnergy.Dsmr.Parser.Service/Application/ConsoleHost.cs:9]: Done!

**Review by sensateiot-cicd — APPROVED** [review]
Approved!

**Pull request merged**